index.html

```diff
@@ -4,9 +4,106 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/index.css">
+    <link rel="stylesheet" href="styles/snow.css">
+    <link rel="stylesheet" href="styles/rain_sun.css">
+    <link rel="stylesheet" href="styles/clouds.css">
+    <link rel="stylesheet" href="styles/select.css">
+    <link rel="stylesheet" href="styles/button.css">
+
+
+
+    <!-- bootstrap for buttons -->
+
+
     <title>Weather Report</title>
 </head>
-<body>
+<body class="default">
     
+    <!-- Header -->
+    <header>
+        <div class="container">
+            <nav id="nav">
+                <img src="https://upload.wikimedia.org/wikipedia/commons/7/7f/Rotating_earth_animated_transparent.gif"/>
+                <h1>Weather Report</h1>
+                <h2>For the lovely city of</h2>
+                <h1 id="city_name">Far Far Away</h1>
```

Comment:
Remember, `h1`, `h2`, etc should be used to form the outline of your document.  

A document should only have a single `h1` tag and the `h2` tag should be used for the title of a section:

```suggestion
                <h1>
                  Weather Report
                  <span>For the lovely city of</span>
                  <span id="city_name">Far Far Away</span>
                </h1>
```